proxy/src/main/java/com/wavefront/agent/listeners/tracing/JaegerThriftCollectorHandler.java

```diff
@@ -93,6 +94,7 @@
   private final ReportableEntityPreprocessor preprocessor;
   private final Sampler sampler;
   private final boolean alwaysSampleErrors;
+  private String proxyLevelApplicationName = "Jaeger";
```

Comment:
mark as final without initializing the value here - do that in a constructor, i.e. 
```
this.proxyLevelApplicationName = StringUtils.isBlank(applicationName) ? "Jaeger" : applicationName.trim();
```